careless-gazelles/Bangazon-Site
# Ar add payment

## Status
Choose:
**READY**

## Description
User is now able add a payment type.
NEW MIGRATIONS AND A SEEDER (don't worry about the seeder right now)

## Steps to Test or Reproduce
update-database
Don't bother seeding the database right now
After logging in, go to your profile, click Add A Payment Method
You should be able to add a new payment type.
